ChaosMod/Effects/db/Player/PlayerZoomZoomCam.cpp

```diff
@@ -0,0 +1,72 @@
+/*
+    Effect by Moxi, based on code from DrUnderscore & ProfessorBiddle
+*/
+
+#include <stdafx.h>
+static Camera zoomCamera = 0;
+float camZoom = 80.f;
+const float camZoomRate = 0.6f;
+int n = 0;
+
+static void UpdateCamera()
+{
+    auto coord = CAM::GET_GAMEPLAY_CAM_COORD();
+    auto rot = CAM::GET_GAMEPLAY_CAM_ROT(2);
+    CAM::SET_CAM_PARAMS(zoomCamera, coord.x, coord.y, coord.z, rot.x, rot.y, rot.z, camZoom, 0, 1, 1, 2);
+}
+
+static void OnStart()
+{
+    zoomCamera = CAM::CREATE_CAM("DEFAULT_SCRIPTED_CAMERA", 1);
+    CAM::RENDER_SCRIPT_CAMS(true, true, 10, 1, 1, 1);
+}
```

Comment:
„CamZoom“ should be reset to the initial 80 here

Reply:
Done